panel/depends.py

```diff
@@ -216,12 +223,24 @@ def eval_fn():
                 fn = eval_function(p)
         return fn
 
-    if iscoroutinefunction(function):
+    if inspect.isasyncgenfunction(function):
```

Comment:
use `from inspect import isasyncgenfunction` instead. You will save an attribute lookup each time this line is run which could be thousands or millions of times.

Reply:
I'll do that but note that this is not inside the core loop and won't be called on each update.